website/docs/artist_hosts_maya_yeti.md

```diff
@@ -44,34 +46,51 @@ You can now publish Yeti cache as any other types. **OpenPype → Publish**. It
 create sequence of `.fur` files and `.fursettings` metadata file with Yeti node
 setting.
 
-### Loading Yeti caches
+:::note Collect Yeti Cache failure
+If you encounter **Collect Yeti Cache** failure during collecting phase, and the error is like
+```fix
+No object matches name: pgYetiMaya1Shape.cbId
+```
+then it is probably caused by scene not being saved before publishing.
+:::
+
+### Loading
 
 You can load Yeti cache by **OpenPype → Load ...**. Select your cache, right+click on
 it and select **Load Yeti cache**. This will create Yeti node in scene and set its
 cache path to point to your published cache files. Note that this Yeti node will
 be named with same name as the one you've used to publish cache. Also notice that
 when you open graph on this Yeti node, all nodes are as they were in publishing node.
 
-### Creating and publishing Yeti Rig
+## Yeti Rigs
```

Comment:
Should we document somewhere that the yetiRig can have any number of `pgYetiMaya` nodes and grooms as it'd like. It doesn't have to be just a single one. The nodes just must be uniquely named, that's all.

Reply:
Done